packages/builder-vite/code-generator-plugin.ts

```diff
@@ -25,16 +25,21 @@ export function codeGeneratorPlugin(options: ExtendedOptions): Plugin {
       // invalidate the whole vite-app.js script on every file change.
       // (this might be a little too aggressive?)
       server.watcher.on('change', (_e) => {
-        const { moduleGraph } = server;
-        const appModule = moduleGraph.getModuleById(virtualFileId);
+        const appModule = server.moduleGraph.getModuleById(virtualFileId);
         if (appModule) {
           server.moduleGraph.invalidateModule(appModule);
         }
-        const storiesModule = moduleGraph.getModuleById(virtualStoriesFile);
+        const storiesModule = server.moduleGraph.getModuleById(virtualStoriesFile);
         if (storiesModule) {
           server.moduleGraph.invalidateModule(storiesModule);
         }
       });
+
+      server.watcher.on('all', (_e, path) => {
```

Comment:
Maybe a comment about why this is being added would be good to have, for anyone else coming across this code later?